test/algorithms/test_min_eigen_optimizer.py

```diff
@@ -168,98 +198,145 @@ def test_converter_list(self):
-        result = min_eigen_optimizer.solve(op)
-        opt_sol = 2
-        self.assertEqual(result.fval, opt_sol)
-        self.assertEqual(len(result.samples), 1)
-        np.testing.assert_array_almost_equal(result.samples[0].x, [0, 1])
         self.assertAlmostEqual(result.samples[0].fval, opt_sol)
-        self.assertAlmostEqual(result.samples[0].probability, 1.0)
-        self.assertEqual(result.samples[0].status, SUCCESS)
-        self.assertEqual(len(result.raw_samples), 1)
-        np.testing.assert_array_almost_equal(result.raw_samples[0].x, [0, 0, 1, 0])
+        self.assertEqual(result.samples[0].status, success)
+        np.testing.assert_array_almost_equal(result.raw_samples[0].x, [1, 0, 0, 0, 0])
         self.assertAlmostEqual(result.raw_samples[0].fval, opt_sol)
-        self.assertAlmostEqual(result.raw_samples[0].probability, 1.0)
-        self.assertEqual(result.raw_samples[0].status, SUCCESS)
-
-        qaoa = QAOA(quantum_instance=quantum_instance)
+        self.assertEqual(result.raw_samples[0].status, success)
+        # test maximize
+        opt_sol = 2
+        qaoa = QAOA(quantum_instance=quantum_instance, reps=2)
         min_eigen_optimizer = MinimumEigenOptimizer(qaoa)
-        result = min_eigen_optimizer.solve(op)
+        result = min_eigen_optimizer.solve(self.op_maximize)
         self.assertEqual(len(result.samples), 8)
         self.assertEqual(len(result.raw_samples), 16)
         self.assertAlmostEqual(sum(s.probability for s in result.samples), 1)
         self.assertAlmostEqual(sum(s.probability for s in result.raw_samples), 1)
         self.assertAlmostEqual(max(s.fval for s in result.samples), 5)
         self.assertAlmostEqual(
-            max(s.fval for s in result.samples if s.status == SUCCESS), opt_sol
+            max(s.fval for s in result.samples if s.status == success), opt_sol
         )
         self.assertAlmostEqual(max(s.fval for s in result.raw_samples), opt_sol)
         for sample in result.raw_samples:
-            self.assertEqual(sample.status, SUCCESS)
-        np.testing.assert_array_almost_equal(result.x, result.samples[0].x)
-        self.assertAlmostEqual(result.fval, result.samples[0].fval)
-        self.assertEqual(result.status, result.samples[0].status)
+            self.assertEqual(sample.status, success)
+        np.testing.assert_array_almost_equal(result.x, [0, 1])
+        self.assertEqual(result.fval, opt_sol)
+        self.assertEqual(result.status, success)
+        np.testing.assert_array_almost_equal(result.samples[0].x, [0, 1])
+        self.assertAlmostEqual(result.samples[0].fval, opt_sol)
+        self.assertEqual(result.samples[0].status, success)
+        np.testing.assert_array_almost_equal(result.raw_samples[0].x, [0, 0, 1, 0])
+        self.assertAlmostEqual(result.raw_samples[0].fval, opt_sol)
+        self.assertEqual(result.raw_samples[0].status, success)
+
+    @data("sv", "qasm")
+    def test_samples_vqe(self, simulator):
```

Comment:
`test_bit_ordering` would be good to be consistent with the comment below, "Test bit ordering"

Reply:
Actually, I changed the comment of `test_samples_vqe` to `"""Test samples for VQE"""`. Since the names of other test cases are `test_samples_qaoa` and `test_samples_numpy_eigen_solver`,  `test_samples_vqe` looks more consistent than `test_bit_ordering`.